custom_components/elasticsearch/es_doc_creator.py

```diff
@@ -29,6 +29,17 @@
 from custom_components.elasticsearch.system_info import SystemInfo
 
 ALLOWED_ATTRIBUTE_TYPES = tuple | dict | set | list | int | float | bool | str | None
+SKIP_ATTRIBUTES = [
+    "friendly_name",
+    "id",
```

Comment:
I haven't seen `id` in my attributes, but I'm not opposed to excluding this if you're finding it pervasive and unhelpful.

Reply:
i probably jumped the gun on this one too